x/ecocredit/basket/simulation/msg_update_basket_fees.go

```diff
@@ -0,0 +1,85 @@
+package simulation
+
+import (
+	"math/rand"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+
+	"github.com/regen-network/regen-ledger/x/ecocredit"
+	basetypes "github.com/regen-network/regen-ledger/x/ecocredit/base/types/v1"
+	types "github.com/regen-network/regen-ledger/x/ecocredit/basket/types/v1"
+	"github.com/regen-network/regen-ledger/x/ecocredit/simulation/utils"
+)
+
+const OpWeightMsgUpdateBasketFees = "op_weight_update_basket_fees" //nolint:gosec
```

Comment:
Here's where golint is failing. It looks like `msg` is missing in the string and the use of `msg` is what requires the lint ignore comment. Updating the string to include `msg` will make the lint error go away.

Reply:
```suggestion
const OpWeightMsgUpdateBasketFees = "op_weight_msg_update_basket_fees" //nolint:gosec
```